modules/gitops-additional-permissions-for-cluster-config.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assembly:
+//
+// * configuring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations.adoc
+
+:_content-type: PROCEDURE
+[id="gitops-additional-permissions-for-cluster-config_{context}"]
+= Providing additional permissions for cluster configuration
+
+For better cluster-configuration management you can add additional permissions to an Argo CD instance. 
```

Comment:
I'm not sure about the perception of "better cluster-configuration management" here. It suggests that out of the box, cluster management capabilities are bad or not good, and the user has to perform additional manual steps to make it better.